velox/experimental/wave/exec/tests/CMakeLists.txt

```diff
@@ -50,9 +93,10 @@ target_link_libraries(
   Boost::regex
   Boost::thread
   Boost::system
-  GTest::gtest
-  GTest::gtest_main
-  GTest::gmock
```

Comment:
Please use the properly name spaced targets.

Reply:
@oerling

Reply:
I have fixed it in https://github.com/facebookincubator/velox/pull/10732